genome_designer/make_new_refs_clean.ipynb

```diff
@@ -0,0 +1,1114 @@
+{
```

Comment:
If you want to commit this, place it in debug and maybe add a date to the file name.